### ISSUE TITLE
No callbacks at first launch of app

### ISSUE THREAD

**Issue by Astervinus** [myflashlab/GameServices-ANE#19]
When I run my app first time, after calling 
```GameServices.google.auth.login();```
I see this image http://take.ms/ozg1r but after this nothing happens. No callbacks triggered.
Then after some timeout I call this method again and see this http://take.ms/IBk41 
and only after that I recieve AuthEvents.LOGIN event and get user info.
Next launches of app works good. 
In my project first call of 
```GameServices.google.auth.login();```
makes as in your demo in this callback:
```actionscript
		private function onSilentTry(e:AuthEvents):void
		{
			if (e.canDoSilentLogin)
			{
				trace("connecting to Game Services, please wait...");
			}else{
				 GameServices.google.auth.login();
			}
		}
```
The second call I make after some timeout if no callbacks triggered. But this is no good.

**Comment by Astervinus** [comment]
And what is the difference between `GameServices.google.auth.login();` and `GameServices.google.auth.getAuth();`?

**Comment by misterPitty** [comment]
Hi. Have you solved this problem?

**Comment by Astervinus** [comment]
Hi. I'm checking login on activate event like this
`
private function handleActivate(e:Event):void
{
	if (!GameServices.google.auth.isLogin) {
		GameServices.google.auth.login();
	}
}
`

**Comment by myflashlab** [comment]
Just released V4.0.0 of this ANE and this ANE now uses the [GoogleSignIn ANE](https://github.com/myflashlab/GoogleSignIn-ANE) to take care of the authentication. Update yourself and make sure to read the new Wiki to see how you should do things. If any problem, please open a new issue because authentication inside the GameServices ANE is no longer relevant.
